setup.py

```diff
@@ -58,9 +60,9 @@ def get_all_extras() -> Dict:
     "jsonschema<4.4.0,>=4.3.0",
     "packaging>=23.1,<24.0",
     "protobuf<4.25.0,>=4.21.6",
-    "pymultihash==0.8.2",
-    "pyyaml==6.0.1",
-    "requests==2.28.1",
+    "pymultihash>=0.8.2",
+    "pyyaml>=6.0.1,<7",
+    "requests>=2.28.1,<3",
```

Comment:
requests version `2.28.2` also has some incompatibly issues with several other dependencies, that's we pinned it to `2.28.1`